packages/types/vite.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'vite';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+	build: {
+		// generate .vite/manifest.json in outDir
+		manifest: true,
+		lib: {
+			entry: './src/main.mts',
+			name: 'pcql-daco-types',
+		},
+		rollupOptions: {
```

Comment:
curious why rollupOptions need to be defined with just an entry point. I think it's covered by lib already?

Reply:
Eh there was some back and forth during setup. This package, I think I chose the wrong Vite template to start with and have since removed these options.